jdaviz/core/template_mixin.py

```diff
@@ -3384,6 +3407,12 @@ def layer_in_spectrum_2d_viewer(data):
                 return True
             return data.label in [l.layer.label for l in self.spectrum_2d_viewer.layers]  # noqa E741
 
+        def layer_in_flux_viewer(data):
+            if not len(self.app.get_viewer_reference_names()):
+                # then this is a bare Application object, so ignore this filter
+                return True
+            return data.label in [l.layer.label for l in self.flux_viewer.layers]  # noqa E741
```

Comment:
`l` is hard to read.
```suggestion
            return data.label in [lyr.layer.label for lyr in self.flux_viewer.layers]  # noqa E741
```